Bottom: a582d774a18656e756c9add470227636876036c4
Top:    da592220eeb434a68bbb6b5ac444be0c433d14f6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 23:59:15 +0000

Refresh of hidgl-fixup-to-use-hid_draw

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 2cf1c91..71dfe6c 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2590,6 +2590,7 @@ ghid_drawing_area_expose_cb (GtkWidget *widget,
   ghid_draw_bg_image ();
 
   common_set_clip_box (priv->hid_draw, &region);
+  /* hid_expose_callback (priv->hid_draw, &region, 0); */
   ghid_draw_everything (&region);
   hidgl_flush_triangles (priv->hid_draw);
